app/src/main/java/org/p2p/wallet/transaction/TransactionManager.kt

```diff
@@ -37,7 +38,7 @@ class TransactionManager private constructor(
     /*
    * This one is to show user transaction id when progress dialog is shown
    * */
-    private val transactionStateFlow = MutableStateFlow<TransactionState>(TransactionState.Progress())
+    private val transactionStateFlow = MutableStateFlow<TransactionState?>(null)
```

Comment:
are we sure we don't want to create a separate state instead of NULL?
Explicit Separate class is always better then NULL

Reply:
I even don't know how name this state x D  None ?

Reply:
None / Initial / Empty / Nothing

Reply:
any synomym that can used instead of NULL. NULL - means absence of something

Reply:
i used null cause null value will not trigger Flow again

Reply:
it's behavior which i wanted to reach to fix this problem x D

Reply:
you can try and filter new separate class like null